install/kubernetes/helm/istio-remote/templates/endpoints.yaml

```diff
@@ -0,0 +1,25 @@
+{{- if .Values.global.remotePilotCreateSvcEndpoint }}
+apiVersion: v1
+kind: Endpoints
+metadata:
+  name: istio-pilot
+  namespace: {{ .Release.Namespace }}
+subsets:
+- addresses:
+  - ip: {{ .Values.global.remotePilotAddress }}
```

Comment:
this won't work with hostnames, right? We're currently using a load balancer name for the pilot address. Would a service with type `ExternalName` work?

Reply:
Eventually we want it to be a DNS name.

Reply:
can we not use one now with an `ExternalName` service instead of selectorless?

Reply:
@Rigdon -- this option is only needed if the remote cluster can't resolve a hostname for istio-pilot.  The whole selectorless service and endpoint creation is optioned on `remotePilotCreateSvcEndpoint`.  If remotePilotAddress is a hostname then you would set `remotePilotCreateSvcEndpoint=false`.

Reply:
Thanks for the clarification. I originally read this as if mTLS required the endpoint be named `istio-pilot.namespace`.